scripts/tomcat/tomcat8/ubuntu/tomcat8_install.sh

```diff
@@ -122,12 +122,12 @@ cp ${SCRIPT_HOME}/etc/default/${TOMCAT_NAME} /etc/default
 
 if ! grep -i --quiet 'ingest/processed' ${TOMCAT_CONFIG_HOME}/server.xml; then
     echo "Adding Tomcat context path for tile images..."
-    sudo sed -i.bak 's@<\/Host>@  <Context docBase=\"'"$HOOT_HOME"'\/ingest\/processed\" path=\"\/static\" \/>\n      &@' ${TOMCAT_CONFIG_HOME}/server.xml
+    sudo sed -i.bak 's@<\/Host>@  <Context docBase=\"'"$HOOT_HOME"'\/userfiles\/ingest\/processed\" path=\"\/static\" \/>\n      &@' ${TOMCAT_CONFIG_HOME}/server.xml
 fi
 
-if ! grep -i --quiet 'allowLinking="true"' ${TOMCAT_CONFIG_HOME}/context.xml; then
+if ! grep -i --quiet '<Resources allowLinking="true" />' ${TOMCAT_CONFIG_HOME}/context.xml; then
     echo "Set allowLinking to true in Tomcat context..."
```

Comment:
This should pull in @sputnam fix from https://github.com/ngageoint/hootenanny-rpms/commit/23cc92bc1c1821e6cc3fb990f14e803494802d15